### ISSUE TITLE
Make Hydrus utilities available for entire test class rather than specific test function.

### ISSUE THREAD

**Issue by mohdomama** [HTTP-APIs/hydrus#129]
### I'm submitting a 
- [x] bug report.
- [ ] feature request.

### Current Behaviour:

As of now, for each test function, the `hydrus`  utilities are set up using the nested `with` statements.
```python
 with set_api_name(self.app, self.API_NAME):
            with set_session(self.app, self.session):
                with set_doc(self.app, self.doc):
                    with self.app.test_client() as client:
```
This is apparently not a good approach because the setting up and destruction of contexts take place for every test function.

### Expected Behaviour:

A better, alternative approach will be to set up contexts once in `setUpClass`, use them throughout the tests and then destroy them in `tearDownClass`. This will allow new developers to focus on writing tests rather than setting up contexts. It will also improve the test run time since setting up and destruction process takes place only once.

### Do you want to work on this issue?

Yes.
I am looking at ways to achieve this. Will send a PR once done.

**Comment by mohdomama** [comment]
On a side note, I think we should add:
```python
import sys
sys.path.insert(0, '')
```
This will explicitly add the cwd to path.This will allow the test to be run as a stand alone script. And prevent the import error which many of the beginners are facing.
Your views @xadahiya ?

**Comment by xadahiya** [comment]
Fixed in #131
